mmedit/models/video_interpolater/cain.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+
+from ..registry import MODELS
+from .basic_interpolater import BasicInterpolater
+
+
+@MODELS.register_module()
+class CAIN(BasicInterpolater):
+    """CAIN model for Video Interpolation.
+
+    Paper: Channel Attention Is All You Need for Video Frame Interpolation
+    Ref repo: https://github.com/myungsub/CAIN
+
+    Args:
+        generator (dict): Config for the generator structure.
+        pixel_loss (dict): Config for pixel-wise loss.
+        train_cfg (dict): Config for training. Default: None.
+        test_cfg (dict): Config for testing. Default: None.
+        pretrained (str): Path for pretrained model. Default: None.
+    """
+
+    def train_step(self, data_batch, optimizer):
+        """Train step.
+
+        Args:
+            data_batch (dict): A batch of data, which requires
+                'inputs' (b, 2, c, h, w), 'target' (b, c, h, w)
+            optimizer (obj): Optimizer.
+
+        Returns:
+            dict: Returned output, which includes:
+                log_vars, num_samples, results (inputs, target and pred).
+
+        """
+        # data
+        inputs = data_batch['inputs']
+        target = data_batch['target']
+
+        # generator
+        pred = self.generator(inputs, padding_flag=False)
```

Comment:
@Yshuo-Li I think the only difference between this function and that in `BasicInterpolator` is this line, where an additional argument `padding_flag=False` is required? If yes, I think you can simply copy the whole function to here, and then modify this line. 

Another way is to set a default value in the backbone. If the default value is False, then this function can be inherited from `BasicInterpolator`. If the default is True. Then `forward_test` can be inherited.